Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -2035,10 +2049,12 @@ class cmGlobalFastbuildGenerator::Detail::Generation
 
 		// Iterate over each configuration
 		// This time to define linker settings for each config
-		for (std::vector<std::string>::iterator iter = context.self->Configurations.begin();
-			iter != context.self->Configurations.end(); ++iter)
+		std::vector<std::string>::const_iterator
+				configIter = context.self->GetConfigurations().begin(),
+				configEnd = context.self->GetConfigurations().end();
+		for (; configIter != configEnd; ++configIter)
 		{
-			std::string configName = *iter;
+			std::string configName = *configIter;
```

Comment:
As you wish. I was just keeping with the previous code (I really was only concerned about the warnings going away)